pkg/vadmin/opts/showrestorepoints/opts.go

```diff
@@ -55,3 +57,39 @@ func WithConfigurationParams(parms map[string]string) Option {
 		s.ConfigurationParams = parms
 	}
 }
+
+func WithArchiveNameFilter(archiveName string) Option {
+	return func(s *Parms) {
+		if s.FilterOptions == nil {
```

Comment:
can we initialize all the fields at the same time as we do in `VShowRestorePointsFactory`?

Reply:
will make sure before we call `Make(opts...)` that `filterOptions` is initialized